open_event/static/js/jquery/jquery.ellipsis.js

```diff
@@ -0,0 +1,43 @@
+// Borrowed from http://stackoverflow.com/a/1022672/1562480
+(function ellipsisInit($) {
+    $.fn.ellipsis = function () {
+        return this.each(function () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650361451/issues/source?bid=3233864&fileBranchId=3324174#l4)